new-lamassu-admin/src/pages/Wizard/components/Wallet/ChooseExchange.js

```diff
@@ -38,7 +38,8 @@ const SAVE_ACCOUNTS = gql`
   }
 `
 
-const isConfigurable = it => R.contains(it)(['kraken', 'itbit', 'bitstamp'])
+const isConfigurable = it =>
+  R.contains(it)(['kraken', 'itbit', 'bitstamp', 'binanceus', 'cex', 'ftx'])
```

Comment:
Perhaps it is worth to move the array with the exchanges to a variable? We could get it from `lib/plugins/common/ccxt.js`, but that is on the back-end side of things and could be troublesome. What to you think?

Reply:
I'll ask Rafael